pkg/server/helpers.go

```diff
@@ -349,3 +352,38 @@ func getUserFromImageUser(user string) (*int64, string) {
 	// If user is a numeric uid.
 	return &uid, ""
 }
+
+// getImageMetadata returns corresponding metadata of the image reference, if image is not
```

Comment:
Sure as long as it's clear that this is a variant of an image pull... and not just a metadata retrieval.. that works!